### DISCUSSION TITLE
Deletion of files once downloaded from s3?

### DISCUSSION THREAD

**Discussion by Hasham04** [vaexio/vaex#1937] [Q&A] [answered]
After I am done transforming and exporting a file that I downloaded from s3, I want to delete the file that was loaded in the data frame. 
The files are stored at user\.vaex\file-cache\s3  and user\.vaex\lock\file-cache, is there a way to delete them from here once I am done.

**Comment by JovanVeljanoski** [comment, marked as answer]
what does "from here" mean? Just delete the files like any other file..

**Reply by Hasham04** [reply]
answered in issue #1939
